test/profile/both.def.exp2

```diff
@@ -0,0 +1,5 @@
+
+FUNCTIONS
+	_Dmain
```

Comment:
Those tabs are generated as part of the profile output and are also in both.exp.